features/storage/FEATURE_STORAGE/cfstore/source/cfstore_test.c

```diff
@@ -374,7 +374,11 @@ int32_t cfstore_test_delete_all(void)
             CFSTORE_ERRLOG("%s:Error: failed to delete key_name=%s, len=%d\r\n", __func__, key_name, (int) len);
             return ret;
         }
-        CFSTORE_HANDLE_SWAP(prev, next);
+        ret = drv->Close(next);
```

Comment:
Hi @bogdanm, this change is required to pass testing. The changes I made in this PR caused [a config store issue](https://github.com/ARMmbed/configuration-store/issues/23) to surface. This is a workaround for that issue.